Pipfile

```diff
@@ -7,11 +7,18 @@ name = "pypi"
 fastapi = {extras = ["all"], version = "*"}
 uvicorn = "*"
 typing = "*"
-pre-commit = "*"
-psycopg2 = "*"
 python-dotenv = "*"
+pydantic = "*"
+sqlalchemy = "*"
 
 [dev-packages]
+fastapi = {extras = ["all"], version = "*"}
```

Comment:
si quiere el pipfile pasa todavia, pero el pipelock nop.

lo que se sube es un readme de how-to y un .txt con las dependecias que lo exportas/creas asi:
pipenv lock -r > requirements.txt